bin/shpec

```diff
@@ -22,7 +22,7 @@ describe() {
 end() {
   : $((test_indent -= 1))
   if [ $test_indent -eq 0 ]; then
-    [ $failures -eq 0 ]
+    [ "$failures" -eq 0 ]
```

Comment:
@locochris this is a weird one; why would the linter complain about this line but not the one above it?